app/build.gradle

```diff
@@ -102,6 +102,7 @@ ext {
 
     androidxLifecycleVersion = '2.3.1'
     androidxRoomVersion = '2.3.0'
+    androidxWorkVersion = '2.5.0'
```

Comment:
So what are we going to do about this? Shouldn't we increase the compile SDK version in this PR?

Reply:
I think we should create a PR right after 0.22.0 is released that just bumps the compile sdk to 31. That's also needed for #7782, so it should be done independently of either of the PRs, which also makes sure the change is revertible (more) easily in case of issues.

Reply:
Done. Merged the change into this PR

Reply:
Updated to 2.7.1